sdk/redis/Microsoft.Azure.Management.Redis/src/Generated/Models/RedisCreateParameters.cs

```diff
@@ -65,12 +65,14 @@ public RedisCreateParameters()
         /// <param name="subnetId">The full resource ID of a subnet in a
         /// virtual network to deploy the Redis cache in. Example format:
         /// /subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/Microsoft.{Network|ClassicNetwork}/VirtualNetworks/vnet1/subnets/subnet1</param>
-        /// <param name="staticIP">Static IP address. Required when deploying a
-        /// Redis cache inside an existing Azure Virtual Network.</param>
+        /// <param name="staticIP">Static IP address. Optionally, may be
+        /// specified when deploying a Redis cache inside an existing Azure
+        /// Virtual Network; auto assigned by default.</param>
         /// <param name="zones">A list of availability zones denoting where the
         /// resource needs to come from.</param>
         /// <param name="tags">Resource tags.</param>
-        public RedisCreateParameters(Sku sku, string location, IDictionary<string, string> redisConfiguration = default(IDictionary<string, string>), string redisVersion = default(string), bool? enableNonSslPort = default(bool?), int? replicasPerMaster = default(int?), int? replicasPerPrimary = default(int?), IDictionary<string, string> tenantSettings = default(IDictionary<string, string>), int? shardCount = default(int?), string minimumTlsVersion = default(string), string publicNetworkAccess = default(string), string subnetId = default(string), string staticIP = default(string), IList<string> zones = default(IList<string>), IDictionary<string, string> tags = default(IDictionary<string, string>))
+        /// <param name="identity">The identity of the resource.</param>
+        public RedisCreateParameters(Sku sku, string location, RedisCommonPropertiesRedisConfiguration redisConfiguration = default(RedisCommonPropertiesRedisConfiguration), string redisVersion = default(string), bool? enableNonSslPort = default(bool?), int? replicasPerMaster = default(int?), int? replicasPerPrimary = default(int?), IDictionary<string, string> tenantSettings = default(IDictionary<string, string>), int? shardCount = default(int?), string minimumTlsVersion = default(string), string publicNetworkAccess = default(string), string subnetId = default(string), string staticIP = default(string), IList<string> zones = default(IList<string>), IDictionary<string, string> tags = default(IDictionary<string, string>), ManagedServiceIdentity identity = default(ManagedServiceIdentity))
```

Comment:
This is a breaking change.  We need to go through breaking change team for the spec PR that generated this change and have them review / approve it.

Reply:
Agree, https://github.com/Azure/azure-rest-api-specs/pull/15828 here didn't have  'Approved-BreakingChange'.

Reply:
@m-nash @ArthurMa1978 
We didn't introduce this change in the above mentioned PR.
It was inherited from the old existing api version.
It looks like this change was created when we tried to fix a s360 swagger correctness issue in our previous api version.
We added missing keys to the above mentioned property in swagger.
https://github.com/Azure/azure-rest-api-specs/pull/15674/files#diff-a42e92e1834992e4831f3839ea16301f631b1a2fb90ca19964cf0347b4323694

Since this is not a breaking change in terms of REST Api, we didn't generate SDK at that time.
We really don't want to modify the above type, it can remain as "IDictionary<string, string>" type.
It is only the SDK generator that was creating this change.

Please advice me on how can we proceed at this stage?

Reply:
I'm fine with this breaking change for Track 1